src/77480.c

```diff
@@ -494,7 +494,7 @@ s32 player_test_move_without_slipping(PlayerStatus* playerStatus, f32* x, f32* y
 
     raycastID = player_raycast_general(0, *x, *y + 0.1, *z, sinTheta, 0, cosTheta, &hitX, &hitY, &hitZ, &hitDepth, &hitNx, &hitNy, &hitNz);
     if (raycastID >= 0 && hitDepth <= depth) {
-        *arg6 = 1;
+        *hasClimbableStep = 1;
```

Comment:
If we know this to be a bool, could you change uses of setting it to be = TRUE / = FALSE?